connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -651,6 +651,7 @@ class SparkConnectPlanner(session: SparkSession) {
         .map(transformExpression)
         .map {
           case x @ UnresolvedAttribute(_) => x
+          case x @ Alias(_, _) => x
```

Comment:
```suggestion
          case ua: UnresolvedAttribute => ua
          case a: Alias => a
```

per https://github.com/databricks/scala-style-guide#pattern-matching

Reply:
Done